distribution/docs/src/main/resources/content/_integrating/_endpoints/crud-endpoints-intro.adoc

```diff
@@ -0,0 +1,15 @@
+:type: endpointIntro
+:status: published
+:title: CRUD Endpoints
+:operations: crud
+:order: 02
+
+== {title}
+
+To perform CRUD (Create, Read, Update, Delete) operations on data or metadata in the catalog, work with one of these endpoints.
```

Comment:
Some endpoints don't support crud operations on resources, can we clearly indicate which endpoints support resources/metadata?

✏️ I thought we referred to the data items in the catalog as resources, since data is a little vague.

Reply:
`Some endpoints don't support crud operations on resources, can we clearly indicate which endpoints support resources/metadata?`

Yes. I'll be coming back through this section for https://github.com/codice/ddf/issues/4374

✏️ I thought we referred to the data items in the catalog as resources, since data is a little vague.

I'd like to get us on a consistent convention for that